backend/lcfs/db/models/NotificationMessage.py

```diff
@@ -0,0 +1,28 @@
+from lcfs.db.base import BaseModel, Auditable
+from sqlalchemy import Column, Integer, ForeignKey, Boolean
+from sqlalchemy.orm import relationship
+
+class NotificationMessage(BaseModel, Auditable):
+    __tablename__ = 'notification_message'
+    __table_args__ = {'comment': "Represents a notification message sent to an application user"}
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+
+    is_read = Column(Boolean, default=False)
+    is_warning = Column(Boolean, default=False)
+    is_error = Column(Boolean, default=False)
+    is_archived = Column(Boolean, default=False)
+
+    origin_user_id = Column(Integer,ForeignKey('user.id'))
+    related_organization_id = Column(Integer, ForeignKey('organization.id'))
```

Comment:
foreign key ids to be updated

Reply:
to keep consistency, i updated the ID keys with prefixes for the organizationa, user and role models.